pkg/metadata/host/host.go

```diff
@@ -217,3 +229,44 @@ func getLogsMeta() *LogsMeta {
 func buildKey(key string) string {
 	return path.Join(common.CachePrefix, packageCachePrefix, key)
 }
+
+func getInstallInfoPath() string {
+	return path.Join(config.FileUsedDir(), "install_info")
+}
+
+func getInstallInfo(infoPath string) (*installInfo, error) {
+	yamlContent, err := ioutil.ReadFile(infoPath)
+
+	if err != nil {
+		return nil, err
+	}
+
+	var install installInfo
+
+	if err := yaml.UnmarshalStrict(yamlContent, &install); err != nil {
+		// file was manipulated and is not relevant to format
+		return nil, err
+	}
+
+	return &install, nil
+}
+
+func getInstallMethod(infoPath string) *InstallMethod {
+	install, err := getInstallInfo(infoPath)
+
+	// if we could not get install info
+	if err != nil {
+		// consider install info as private
+		return &InstallMethod{
+			Name:    "private",
```

Comment:
```suggestion
			Name:    "undef",
```
or "unspecified" or something.

Reply:
👍